Gemfile

```diff
@@ -2,3 +2,5 @@ source 'https://rubygems.org'
 
 # Specify your gem's dependencies in google-authenticator.gemspec
 gemspec
+
+gem "sqlite3", "~> 1.3.6"
```

Comment:
What's up with adding sqlite here?

Reply:
had some gem compatibility issue without this line, i can remove